__std__/cells/plutus-apps/packages/combined-plutus-apps-haddock.nix

```diff
@@ -24,11 +24,7 @@ cell.library.combine-haddock {
 
       == Handy module entrypoints
 
-        * "PlutusTx": Compiling Haskell to PLC (Plutus Core; on-chain code).
-        * "PlutusTx.Prelude": Haskell prelude replacement compatible with PLC.
         * "Plutus.Contract": Writing Plutus apps (off-chain code).
-        * "Ledger.Constraints": Constructing and validating Plutus
```

Comment:
`module Ledger.Constraints` is nowhere to be found in `plutus` nor `plutus-apps` repos?

Reply:
It's replaced by `Ledger.Tx.Constraints`.